infrastructure/modules/worker/main.tf

```diff
@@ -1,5 +1,9 @@
+locals {
+  deployment_service_name = var.deployment_service_name == "" ? var.name : var.deployment_service_name
```

Comment:
Is there a reason to not just make this compulsory? I sometimes get lost where things get magically created, and have to go digging them out.

Reply:
This lets you override the default service name - which is necessary where the service name will not match an expected static pattern (catalogue-DATE services are an example of this).

Reply:
Ah ok, so it's expected the name is what we want, and this is to override.

`deployment_service_name_override` or a comment?  I would feel like I should give this if coming in fresh